e2e/pages/adf/demo-shell/tasksCloudDemoPage.ts

```diff
@@ -51,4 +53,29 @@ export class TasksCloudDemoPage {
         Util.waitUntilElementIsVisible(this.activeFilter);
         return this.activeFilter.getText();
     }
+
+    createNewTask() {
```

Comment:
The name of this method makes me think this method is creating a task. Maybe you could rename it to something that indicates that the method is just clicking on two buttons in order to open the start task dialog.